ios/FluentUI/Controls/ButtonVnext.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+public class ButtonVnextState: ObservableObject {
+	@Published var title: String = ""
+	@Published var isDisabled: Bool = false
+}
+
+public struct ButtonVnextViewButtonStyle: ButtonStyle {
+	var targetButton: ButtonVnextView
+
+	public init(targetButton: ButtonVnextView) {
+		self.targetButton = targetButton
+	}
+
+	public func makeBody(configuration: Self.Configuration) -> some View {
+		configuration.label
+			.foregroundColor(Color(targetButton.state.isDisabled ?
+									targetButton.tokens.disabledTitleColor :
+									(configuration.isPressed ?
+										targetButton.tokens.highlightedTitleColor :
+										targetButton.tokens.titleColor)))
```

Comment:
How will a token like this be aware of context (say it's primary color and needs to be green for some views but blue for other views)?

Reply:
I'm working on adding the API changes on the auto generated code to allow overrides of tokens based on windows.
I'll take care of that in a separate PR.